torch/_inductor/codegen/rocm/rocm_kernel.py

```diff
@@ -210,6 +213,34 @@ def size(
         val = sympy_product(sizes)
         return cexpr(self.rename_indexing(val))
 
+    def contiguous_stride(self, node: Optional[IRNode], default_value: int = 0) -> str:
+        """
+        Hook called from template call to get the contiguous stride of an arg.
```

Comment:
Could you elaborate on what ```contiguous stride``` means? For example, does it refer to the stride of a particular dimension? It would be great if we could have some example to explain what it stands for. Thanks.

Reply:
Well the motivation here is, having a torch tensor metadata, provide the stride parameter which CK template expects when constructing the gemm argument. Which is the size of the contiguous dimension in all cases except for the broadcasted bias (either column and row wise) where it is 0.